tests/cmake_name_conflict/enc/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) Open Enclave SDK contributors.
+# Licensed under the MIT License.
+
+oeedl_file(../name_conflict.edl enclave gen)
```

Comment:
Please do not use `oeedl_file` in new code. This script is deprecated.

Reply:
Could you check if the updated modifications works for you? I'm also wondering if we need to refactor the use of oeedl_fiile in all of our tests.

Reply:
Yes, that is work that should be done.